ONSdigital/sdc-responses-dashboard
# adding ignore

# What and why?
This PR updates a few pipfile dependencies and adds an ignore to the makefile.

# How to test?
Deploy and run tests.

Do pipenv install, pipenv graph, and make sure the following dependency versions are correct:

flask-cors = “>=4.0.2”
certifi = “>=2024.07.04”
urllib3 = “>=2.2.2”

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1315)
